lib/zendesk_apps_tools/deploy.rb

```diff
@@ -8,19 +8,20 @@ def deploy_app(connection_method, url, body)
       response = connection.send(connection_method) do |req|
         req.url url
         req.headers[:content_type] = 'application/json'
-
         req.body = JSON.generate body
       end
 
       check_status response
 
     rescue Faraday::Error::ClientError => e
       say_error_and_exit e.message
+    rescue JSON::ParserError => e
```

Comment:
I believe:
```ruby
rescue Faraday::Error::ClientError, JSON::ParserError => e
  say_error_and_exit e.message
end
```

would be more concise, but may invalidate the reason for this PR?